functions.py

```diff
@@ -0,0 +1,63 @@
+import time
+from objects import *
+
+
+# score_a = 0
+# score_b = 0
+
+# rockets movement
+
+def rocket_left_up():
+    y = rocket_left.ycor()
+    if y < 250:
+        y += 20
+        rocket_left.sety(y)
+
+
+def rocket_left_down():
+    y = rocket_left.ycor()
+    if y > -250:
+        y -= 20
+        rocket_left.sety(y)
+
+
+def rocket_right_up():
+    x = rocket_right.ycor()
+    if x < 250:
+        x += 20
+        rocket_right.sety(x)
+
+
+def rocket_right_down():
+    x = rocket_right.ycor()
+    if x > - 250:
+        x -= 20
+        rocket_right.sety(x)
+
+
+# ball movement
+
+def win_side(score_a, score_b):
```

Comment:
Start function and method names from the verb. In other words, rename this method to, say, `get_winner`.